### ISSUE TITLE
Pack "lyte.lua" with the release binaries

### ISSUE THREAD

**Issue by morew4rd** [lyte2d/lyte2d#57]
Also document it, maybe?

VSCode lua extension is pretty cool and the experience getting code completion with types is pretty amazing.

**Comment by Zorbn** [comment]
Agreed, it would be nice to have it in the releases and maybe have a little text somewhere about how to use it with VSCode at least. Currently I assume only people who have built Lyte know about `lyte.lua`, but probably most people would want to know about it.

eg.
In `.vscode/settings.json`:
```json
{
    "Lua.workspace.library": [
        "path/in/workspace/to/lyte.lua"
    ],
    "Lua.runtime.version": "Lua 5.1"
}

```

**Comment by morew4rd** [comment]
done for the next release (0.7.14)

**Comment by morew4rd** [comment]
Should be documented: https://github.com/lyte2d/lyte2d/issues/79
